npm/webpack-preprocessor/index.ts

```diff
@@ -1,19 +1,33 @@
+import Bluebird from 'bluebird'
+import Debug from 'debug'
+import _ from 'lodash'
+import * as events from 'events'
+import * as path from 'path'
+import webpack from 'webpack'
+import utils from './lib/utils'
+import { crossOriginCallbackStore } from './lib/cross-origin-callback-store'
 import { overrideSourceMaps } from './lib/typescript-overrides'
+import { compileCrossOriginCallbackFiles } from './lib/cross-origin-callback-compile'
 
-import * as Promise from 'bluebird'
```

Comment:
+1 for being explicit about if it's a regular or a bluebird promise

Reply:
Plays better with typescript this way too, since it's not shadowing the global 'Promise'